lib/ladybug/energy_model/model.rb

```diff
@@ -68,7 +68,7 @@ def initialize(hash)
 
       # check if the model is valid
       def valid?
-        JSON::Validator.validate(@hash, @@schema)
+        return validation_errors.empty?
       end
 
       # return detailed model validation errors
```

Comment:
I think we will need to add a new method like `create_schedules` and call it in `create_openstudio_objects` to actually import the schedules from the JSON

Reply:
#4